vendor/bat-native-ads/src/bat/ads/internal/tokens/redeem_unblinded_token/create_confirmation_util.cc

```diff
@@ -84,11 +84,11 @@ std::string CreateConfirmationRequestDTO(
     const std::string locale =
         brave_l10n::LocaleHelper::GetInstance()->GetLocale();
 
-    if (locale::HasLargeAnonymity(locale)) {
+    if (locale::IsMemberOfAnonymitySet(locale)) {
       const std::string country_code = brave_l10n::GetCountryCode(locale);
       dto.SetKey("countryCode", base::Value(country_code));
     } else {
-      if (locale::IsAnonymous(locale)) {
+      if (locale::ShouldClassifyAsOther(locale)) {
```

Comment:
@tmancey if a country is not part of the anonymity set nor classified as other we don't send a `country` key at all - is this still intended behaviour?

Reply:
This is still intended behaviour